lib/asciidoctor/converter/factory.rb

```diff
@@ -44,14 +44,14 @@ class << self
         # Returns the default [Factory] singleton instance
         def default initialize_singleton = true
           return @__default__ || new unless initialize_singleton
-          # FIXME this assignment is not thread_safe, may need to use a ::Threadsafe helper here
+          # FIXME this assignment is not concurrent-ruby, may need to use a ::Threadsafe helper here
```

Comment:
i think we really want to say that the assignment is not "thread safe" right ?

Reply:
right you are!